vapi.ts

```diff
@@ -145,14 +145,22 @@ export default class Vapi extends VapiEventEmitter {
       if (this.call) {
         this.cleanup();
       }
+      const isVideoRecordingEnabled =
+        webCall.assistantOverrides?.artifactPlan?.videoRecordingEnabled ??
+        webCall.assistant?.artifactPlan?.videoRecordingEnabled ??
```

Comment:
won't work for assistantId

Reply:
Now sending webCall?.artifactPlan?.videoRecordingEnabled as part of response of call/web so the backend controls all the scenarios